hackathon_site/dashboard/frontend/src/slices/event/teamSlice.ts

```diff
@@ -107,3 +108,10 @@ export const teamMemberNamesSelector = createSelector(
             (member) => member.user.first_name + " " + member.user.last_name
         )
 );
+
+export const teamSizeValidSelector = createSelector([teamSliceSelector], (teamSlice) =>
```

Comment:
Hi Ethan, so so sorry about this, but I realized we do need the team size somewhere else, could you change this selector back to how you had it before? 🙏🙏🙏

And in CartSummary.tsx could you do the minmax logic in a constant instead of directly in the jsx